packages/node_modules/@webex/internal-plugin-devices/test/unit/spec/devices.js

```diff
@@ -156,7 +156,15 @@ describe('plugin-devices', () => {
 
     describe('#serialize()', () => {
       it('should serialize features in a format compatible with wdm', () => {
-        assert.deepEqual(devices.serialize().features, dto.features);
+        assert.hasAllKeys(
+          devices.serialize().features.entitlement,
+          Object.keys(dto.features.entitlement)
+        );
+
+        assert.hasAllKeys(
```

Comment:
Would it be beneficial to split this into another test of `should serialize user features keys`?

Reply:
This has been split.